core/modules/field/modules/options/options.element.inc

```diff
@@ -0,0 +1,384 @@
+<?php
+
+/**
+ * @file
+ * All logic for options form elements.
+ */
+
+/**
+ * Logic function for form_options_expand(). Do not call directly.
+ *
+ * @see form_options_expand()
+ */
+function _form_options_expand($element) {
+  $element['#options'] = isset($element['#options']) ? $element['#options'] : array();
+  $element['#multiple'] = isset($element['#multiple']) ? $element['#multiple'] : FALSE;
+
+  $element['#tree'] = TRUE;
+  $element['#theme'] = 'options';
+
+  $path = backdrop_get_path('module', 'options');
+  $element['#attached']['js'] = array(
+    'core/misc/tabledrag.js' => array('group' => JS_LIBRARY, 'weight' => 5),
+    'core/misc/jquery.cookie.js' => array('group' => JS_LIBRARY),
+    $path . '/js/options.js',
+  );
+  $element['#attached']['css'] = array(
+    $path . '/css/options.css',
+  );
```

Comment:
Seems like all of this should be combined into a library in `hook_library_info()` and attached all together as a single library.

Reply:
done